Sources/Basic/FixableError.swift

```diff
@@ -0,0 +1,23 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright 2015 - 2016 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+public protocol FixableError: CustomStringConvertible {
```

Comment:
Ideally it would be put in the package related to either building or cli commands. However that would result in a circular dependency, as lower-level packages need access to this protocol. Any suggestions for another module?